### ISSUE TITLE
Update Wiki Documentation for Example Stats and Abilities to match updated class_prefabs.c

### ISSUE THREAD

**Issue by katie-gao** [uchicago-cs/chiventure#1372]
Van and I are going to update the wiki documentation for the new example stats and abilities. As seen from issue #1276, we have updated rogue, wizard, bard, ranger, and monk. Now, we want to change the wiki documentation to have the abilities that we added, their stats, and summaries.

**Comment by katie-gao** [comment]
Van and I have updated the wiki documentation for the new example stats and abilities for the player classes that correspond to the new code we wrote in class_prefabs.c Here is the link to the wiki page: https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Example-Stats-and-Abilities.

**Comment by katie-gao** [comment]
Van and I have updated the wiki documentation for the new example stats and abilities for the player classes that correspond to the new code we wrote in class_prefabs.c Here is the link to the wiki page: https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Example-Stats-and-Abilities. So there were changes in the rogue, bard, wizard, ranger, and monk player classes combat abilities to match what was written in class_prefabs.c in the playerclass module. All of the damages and names are now corresponding that of the code. Now that all of the changes made in the code or put in the documentation, we are closing this issue.
